client/state/current-user/email-verification/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { createReducer } from 'state/utils';
+// import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_REQUEST_SUCCESS,
+	EMAIL_VERIFY_REQUEST_FAILURE,
+	EMAIL_VERIFY_STATE_RESET,
+} from 'state/action-types';
+
+export default createReducer( {}, {
+	[ EMAIL_VERIFY_REQUEST ]: () => ( {
+		pendingRequest: true,
+		emailSent: false,
+		error: false,
```

Comment:
Cool - thanks for the suggestion. I was trying not to break up the reducers into too many pieces since there's a lot of boilerplate to go through in that case, but can see a case for it either way 👍